lib/screens/app/explorer/overview.dart

```diff
@@ -1,54 +1,195 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:seeds/constants/app_colors.dart';
+import 'package:seeds/providers/notifiers/balance_notifier.dart';
+import 'package:seeds/providers/notifiers/planted_notifier.dart';
+import 'package:seeds/providers/notifiers/voice_notifier.dart';
 import 'package:seeds/providers/services/navigation_service.dart';
 import 'package:seeds/widgets/main_card.dart';
-// import 'package:seeds/widgets/seeds_button.dart';
+import 'package:shimmer/shimmer.dart';
+import 'package:seeds/utils/string_extension.dart';
 
-class Overview extends StatelessWidget {
+class Overview extends StatefulWidget {
   const Overview({
     Key key,
   }) : super(key: key);
 
-  Widget buildCategory(String title, String iconName, Function onTap) {
-    return MainCard(
-      onPressed: onTap,
-      margin: EdgeInsets.only(bottom: 10),
-      padding: EdgeInsets.all(15),
-      child: Row(
-        children: <Widget>[
-          Container(
-            width: 28,
-            height: 28,
-            margin: EdgeInsets.only(right: 15),
-            child: Image(
-              image: AssetImage(iconName),
+  @override
+  _OverviewState createState() => _OverviewState();
+}
+
+class _OverviewState extends State<Overview> {
+  Widget buildCategory(
+    String title,
+    String subtitle,
+    String iconName,
+    String balanceTitle,
+    String balanceValue,
+    Function onTap,
+  ) {
+    var width = MediaQuery.of(context).size.width;
+
+    return Row(
+      children: [
+        Container(
+          width: width * 0.58,
+          child: InkWell(
```

Comment:
Use onPressed function in `MainCard` instead of Inkwell. Because inkwell as widget parent will create the ripple animation effect below the widget. There is already one inkwell in main card you can use it using `onPressed`

Reply:
Correct